mirrord/kube/src/api/kubernetes.rs

```diff
@@ -178,7 +178,14 @@ impl KubernetesAPI {
         let pod_ips = runtime_data
             .as_ref()
             .filter(|runtime_data| !runtime_data.pod_ips.is_empty())
-            .map(|runtime_data| runtime_data.pod_ips.join(","));
```

Comment:
Now what if they're empty? :eyes:

Reply:
it's same as before, no? empty string. @DmitryDodzin did you consider this? should I worry/fix?

Reply:
It should be the same because of the .filter above